subprojects/docs/src/docs/release/notes.md

```diff
@@ -387,13 +387,14 @@ In Gradle 7.0 we moved the following classes or methods out of incubation phase.
         - org.gradle.tooling.events.test.Destination
         - org.gradle.tooling.events.test.TestOutputDescriptor
         - org.gradle.tooling.events.test.TestOutputEvent
+    - Debugging
+      - org.gradle.tooling.TestLauncher.withTaskAndTestClasses(String, Iterable)
```

Comment:
I also meant to indent these lines more, since the indentation is now inconsistent with the other section. Does this render correctly?